src/gpu/generic/sycl/ref_deconvolution.hpp

```diff
@@ -59,7 +59,8 @@ struct ref_deconvolution_bwd_weights_t
                             data_d, diff_weights_d, diff_dst_d)
                     && attr()->has_default_values(sm::scales_runtime
                             | sm::zero_points_runtime | sm::post_ops
-                            | sm::sum_dt);
+                            | sm::sum_dt)
```

Comment:
(not related to this PR) there is no scaling for bwd pass.

Reply:
fixed, along with post ops